Analyser/AmplFilter/src/com/Main.java

```diff
@@ -0,0 +1,87 @@
+package com.roboy;
```

Comment:
for the future: create a new branch with the name of your feature and/or your name
`git checkout -b petr/your_feature`